src/DownloadTrait.php

```diff
@@ -48,7 +48,11 @@ public function downloadAllProviders(OutputInterface $output, bool $downloadImag
     {
         $vcards = [];
 
-        foreach ($this->config['local'] as $file) {
+        $localConfig = $this->config['local'] ?? [];
+        if (!$localConfig) {
+            error_log('Notice: Local config is empty and will not be applied');
```

Comment:
Why do we need the error message?

Reply:
I think this message will useful for cases when no local config passed, because there is no error anymore.